201901_gaussian_fg_lensed_cmb_realistic_noise/simulation.cfg

```diff
@@ -0,0 +1,77 @@
+
+output_nside = 4096
+output_folder = output/$output_nside
+channels = all
+unit = uK_CMB
+mapsims_version = "0.1"
+so_pysm_models_version = "0.1"
+
+[ other_components ]
+
+    [[ noise ]]
+
+    class = mapsims.SONoiseSimulator
+    nside = $output_nside
+    return_uK_CMB = True
+    sensitivity_mode = "baseline"
+    apply_beam_correction = True
+    apply_kludge_correction = True
+    scanning_strategy = "classical"
+    LA_number_LF = 1
+    LA_number_MF = 4
+    LA_number_UHF = 2
+    SA_years_LF = 1
+    SA_one_over_f_mode = "pessimistic"
+    seed = 8974
+
+[ pysm_components ]
+
+    # Remove this if no default PySM components are needed
+    pysm_components_string = "a1"
```

Comment:
`a1` was just for testing purposes, do we need any of the PySM components? https://pysm-public.readthedocs.io/en/latest/models.html

Reply:
I wouldn't add any PySM component in the first run with Gaussian FG. I'm working to select the models of PySM for the first set of realistic sims.
I would do different sets with increasingly higher complications. With the first one having synch+dust+ame in polarization and also free-free in temperature. No spatial variation of indices, single dust component, no curvature for synch.